spring-batch-infrastructure/src/main/java/org/springframework/batch/item/file/FlatFileItemWriter.java

```diff
@@ -39,11 +40,15 @@
  * @author Dave Syer
  * @author Michael Minella
  * @author Mahmoud Ben Hassine
+ * @author Glenn Renfro
  */
 public class FlatFileItemWriter<T> extends AbstractFileItemWriter<T> {
 
 	protected LineAggregator<T> lineAggregator;
 
+	// default encoding for writing to flat files - set to charset of this Java virtual machine.
+	public static final String DEFAULT_CHARSET = Charset.defaultCharset().name();
```

Comment:
There is already a DEFAULT_CHARSET constant in [AbstractFileItemWriter](https://github.com/spring-projects/spring-batch/blob/d8fc58338d3b059b67b5f777adc132d2564d7402/spring-batch-infrastructure/src/main/java/org/springframework/batch/item/support/AbstractFileItemWriter.java#L75). It is that value that should be changed instead of adding a new constant here.